.github/workflows/rust-docker-build.yml

```diff
@@ -38,6 +38,9 @@ jobs:
                     - image: cymbal
                       dockerfile: ./rust/Dockerfile
                       project: 8dq0xkk0ck
+                    - image: et-embedding-worker
```

Comment:
I feel like `et` is ambiguous to people who aren't us, why not just use `error-tracking`?